mlflow/ml-package-versions.yml

```diff
@@ -351,17 +351,12 @@ spark:
       ">= 0.0.0": ["scikit-learn"]
       "<= 3.3.3": ["pandas<2"]
     run: |
-      find tests/spark/autologging/ml -name 'test*.py' | xargs -L 1 pytest
-
-      # TODO: Fix spark datasource autologging and test it against non-preview versions of pyspark
-      if [ "$(pip show pyspark | grep Version | cut -d' ' -f2)" = "3.0.0" ]; then
-        YELLOW='\033[33;1;4m'
-        NO_COLOR='\033[0m'
-        echo -e "${YELLOW}WARNING: Spark datasource autologging currently only works for Spark 3.0.0-preview.${NO_COLOR}"
-        pip uninstall -y pyspark
-        ./dev/setup-spark-datasource-autologging.sh
-        find tests/spark/autologging/datasource -name 'test*.py' | xargs -L 1 pytest
-      fi
+      # Build Java package
+      pushd mlflow/java/spark
+      mvn package -DskipTests -q
```

Comment:
I think we need to use `mvn install ...` if we want to run test in following script ?